### PR TITLE
net-dns/knot: version bump

### PR DESCRIPTION
**Pull request by nemunaire**
Hi!

Here is a new knot minor release!

### CODE REVIEW

File: net-dns/knot/knot-2.6.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	"rosedb"
+	"+rrl"
+	"+stats"
+	"+synthrecord"
+	"+whoami"
+)
+IUSE="doc caps +fastparser idn libidn2 systemd +utils ${KNOT_MODULES[@]}"
+
+RDEPEND="
+	>=dev-db/lmdb-0.9.15
```
Comment (monsieurp) [inline net-dns/knot/knot-2.6.6.ebuild]: This version is no longer in the tree. Please update this line.

File: net-dns/knot/knot-2.6.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	dev-libs/libedit
+	>=dev-libs/userspace-rcu-0.5.4
```
Comment (monsieurp) [inline net-dns/knot/knot-2.6.6.ebuild]: Ditto.

File: net-dns/knot/knot-2.6.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+	dev-python/lmdb
+	>=net-libs/gnutls-3.3:=
+	caps? ( >=sys-libs/libcap-ng-0.6.4 )
```
Comment (monsieurp) [inline net-dns/knot/knot-2.6.6.ebuild]: Same.

**Comment by nemunaire** [comment]
Updated! All versions in ebuild, mostly picked from `configure.ac`, are no more in the tree. Just kept the dependencies without version.

File: net-dns/knot/knot-2.6.6.ebuild [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+	doc? ( dev-python/sphinx )
+"
+
+S="${WORKDIR}/${P/_/-}"
+
+src_configure() {
+	local u
+	local my_conf=()
+	for u in "${KNOT_MODULES[@]#+}"; do
+		my_conf+=("$(use_with $u module-$u)")
```
Comment (mgorny) [inline net-dns/knot/knot-2.6.6.ebuild]: Please always use `${x}` instead of `$x`.

File: net-dns/knot/knot-2.6.6.ebuild [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+	done
+
+	econf \
+		--with-storage="${EPREFIX}/var/lib/${PN}" \
```
Comment (mgorny) [inline net-dns/knot/knot-2.6.6.ebuild]: Why not include all of them in `my_conf`? That would save us from the backslash hell.

**Comment by nemunaire** [comment]
Updated, following @mgorny remarks.

**Pull request closed without merging**